app/views/admin/manage_users.php

```diff
@@ -15,7 +15,9 @@
         <?php foreach ($users as $user): ?>
             <tr class="users-rows">
                 <td>
-                    <a href="/profile/<?= htmlspecialchars($user->getUsername()); ?>" class="view-profile">View Profile</a>
+                    <a href="/profile/<?= htmlspecialchars($user->getUsername()); ?>" class="view-profile">
+                        <img class="" alt="">
+                    </a>
```

Comment:
no class and no alt

Reply:
The class an alt are implemented through javascript.

As seen upon inspecting the page, profile pictures do have class and alt text.